<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CONFIGURAÇÃO DAS AULAS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(28, 30, 35);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_cancelar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49,54,65);
font: 87 10pt &quot;Arial Black&quot;;
color: rgb(94,141,222);</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(28, 30, 35);
font: 87 13pt &quot;Arial Black&quot;;
color: rgb(94,141,222);</string>
   </property>
   <property name="text">
    <string>Que aulas você tem na segunda?</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_1aula">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49,54,65);
font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>1° AULA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2aula">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49,54,65);
font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>2° AULA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3aula">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49,54,65);
font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>3° AULA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4aula">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49,54,65);
font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>4° AULA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5aula">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49,54,65);
font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>5° AULA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6aula">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49,54,65);
font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>6° AULA(OPCIONAL)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Button_continuar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49,54,65);
font: 87 10pt &quot;Arial Black&quot;;
color: rgb(94,141,222);</string>
   </property>
   <property name="text">
    <string>CONTINUAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
